tests/vector/test_vector_env.py

```diff
@@ -1,7 +1,11 @@
+from functools import partial
+from typing import Callable, Type
+
 import numpy as np
 import pytest
 
-from gym.spaces import Tuple
+from gym import Space, spaces
```

Comment:
Could you remove spaces as I dont think you should need to import the module and add to the following line

Reply:
fixed in 49ee20904ac3a4a1dba3020d1ebd11076848f376